python/afdko/beztools.py

```diff
@@ -1207,12 +1207,14 @@ def saveChanges(self):
 					tf.write(data)
 					tf.close()
 
-			elif  fontType == 2: # PS.
+			elif  2 <= fontType <= 3: # PS (PFA or PFB)
```

Comment:
would this be better?
```python
elif fontType in (2, 3):
```

Reply:
Yes, that expresses the intent better, good suggestion. I updated in https://github.com/adobe-type-tools/afdko/pull/1011/commits/37a733c0a18267395d411761567573a820f4886a